src/Diagnostics.RuntimeHost/Startup.cs

```diff
@@ -52,14 +52,9 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<IAssemblyCacheService, AssemblyCacheService>();
 
             bool searchIsEnabled = false;
-            if (Environment.IsProduction())
-            {
-                searchIsEnabled = Convert.ToBoolean(Registry.GetValue(RegistryConstants.SearchAPIRegistryPath, RegistryConstants.SearchAPIEnabledKey, string.Empty));
-            }
-            else
-            {
-                searchIsEnabled = Convert.ToBoolean(Configuration[$"SearchAPI:{RegistryConstants.SearchAPIEnabledKey}"]);
-            }
+            
+            searchIsEnabled = Convert.ToBoolean(Configuration[$"SearchAPI:{RegistryConstants.SearchAPIEnabledKey}"]);
```

Comment:
This should be a `TryParse` style operation if you're not checking for `FormatException`